### DISCUSSION TITLE
Switch to role table

### DISCUSSION THREAD

**Discussion by hernandohack** [spatie/laravel-permission#1698] [Q&A] [answered]
As a change so that it doesn't point to the new roles table.
I already have a "rol" table, how do I make it point to "rol" and not to "roles"

Thank you.

**Comment by drbyte** [comment, marked as answer]
Extend the Role model and define the table name there.
https://laravel.com/docs/master/eloquent#table-names

**Comment by hernandohack** [comment]
Thank you so much.
